versions.tf

```
@@ -1,5 +1,5 @@
 /**
- * Copyright 2021 Google LLC
+ * Copyright 2022 Google LLC
```

please keep original date
```suggestion
 * Copyright 2018-2023 Google LLC
```

---

Addressed.